frontend/src/theme/theme.tsx

```diff
@@ -34,6 +34,7 @@ export const getDesignTokens: (mode: ThemeModeType, isRTL: boolean) => ThemeOpti
       h6: {
         fontSize: '0.875rem',
         fontWeight: 600,
+        color: '#96989D',
```

Comment:
This affects all h6 elements, which is not what we want. For example, the selected element in the sidebar should be white.

Reply:
Yes, I believe that doing so would be the cleanest implementation.